src/rgw/driver/rados/rgw_cr_rados.cc

```diff
@@ -833,9 +833,20 @@ int RGWAsyncFetchRemoteObj::_send_request(const DoutPrefixProvider *dpp)
   } else {
       // r >= 0
       if (bytes_transferred) {
-        send_sync_notification(
-            dpp, store, &dest_bucket, &dest_obj, attrs, *bytes_transferred,
-            {rgw::notify::ObjectSyncedCreate, rgw::notify::ReplicationCreate});
+        // bucket attrs are required for notification and since its not loaded,
+        // reload the bucket
+        int ret = dest_bucket.load_bucket(dpp, null_yield);
```

Comment:
since this is required for `send_sync_notification`, move the `load_bucket` into that function so that any future calls to that function will account for this use case